rust/cubesqlplanner/cubesqlplanner/src/planner/planners/multi_stage_query_planner.rs

```diff
@@ -376,7 +395,7 @@ impl MultiStageQueryPlanner {
 
         let childs = member_childs(&member)?;
 
-        let description = if childs.is_empty() {
+        let description = if childs.is_empty() || !has_multi_stage_members(&member, false)? {
             if has_multi_stage_members(&member, false)? {
```

Comment:
is it correct to check the same condition `has_multi_stage_members(&member, false)` twice?

Reply:
fixed